spec/javascript/components/molecules/__snapshots__/LeaseUpSidebar.test.js.snap

```diff
@@ -55,10 +57,11 @@ exports[`LeaseUpSidebar should render with empty status items correctly 1`] = `
     className="show-large-up"
   >
     <LeaseUpSidebarButtons
+      isLoading={false}
       onAddCommentClicked={null}
       onChangeStatus={null}
       onSaveClicked={null}
-      status="Approved"
+      status={null}
```

Comment:
Ah this is because we aren't explicitly passing in currentStatus anymore, we infer it from the status items that are passed in.

I updated the test file to expect null in the cast where no statusItems are present (my understanding is this shouldn't happen in the real world) and to follow the testing standard we came up with. Now there is only one snapshot test, and otherwise we manually test the features for that component.